__tests__/app.test.js

```diff
@@ -35,5 +35,39 @@ describe("NCNews-Server Unit Tests", () => {
     
     });
 
+    describe("GET /api/articles/", () => {
+
+        test("Responds with an array of objects, where the objects have the following 7 properties: author, title, article_id, topic, created_at, votes, comment_count", () => {
+
+            return request(app).get("/api/articles").expect(200).then((response) => {
+                const articlesArray = response.body.articles;
+
+                articlesArray.forEach((articleObject) => {
+                    expect(articleObject).toEqual(
+                        expect.objectContaining({
+                            author: expect.any(String),
+                            title: expect.any(String),
+                            topic: expect.any(String),
+                            article_id: expect.any(Number),
+                            created_at: expect.any(String),
+                            votes: expect.any(Number),
+                            comment_count: expect.any(Number)
+                    }));
+                });
+            });
+        });
+
+        test("Responds with an array of objects, where the objects are sorted by Date using the 'created_at' property in Descending order (latest dates first)", () => {
```

Comment:
Just checking the first two isn't a stringent way of checking the order, as the rest could all be out of order. Look at using jest-sorted to do this